system-tests/tests/smoke/cre/por_test.go

```diff
@@ -444,46 +451,46 @@ func setupPoRTestEnvironment(
 		// create CRE CLI settings file
 		var settingsErr error
 		creCLISettingsFile, settingsErr = libcrecli.PrepareCRECLISettingsFile(
-			universalSetupOutput.BlockchainOutput.SethClient.MustGetRootKeyAddress(),
+			homeChainOutput.SethClient.MustGetRootKeyAddress(),
```

Comment:
here things get tricky... what if we want to deploy the DF cache on non-home chain?